### DISCUSSION TITLE
why BaseUrl is not exposed as a fluid value?

### DISCUSSION THREAD

**Discussion by lampersky** [OrchardCMS/OrchardCore#8971] [Q&A] [answered]
I've just upgraded to the latest OC version (1.0.0-rc2-16010) and now I can't get access to **BaseUrl** from within Liquid.

This is not working anymore:
`{{Site.BaseUrl | raw}}`

I've checked current source code and why this line:
`nameof(ISite.BaseUrl) => new StringValue(site.BaseUrl),`
is missing here?

How can I read BaseUrl?

![image](https://user-images.githubusercontent.com/6403130/112514359-840cb180-8d95-11eb-82f4-9fcd05b8d396.png)

**Comment by jtkech** [comment, marked as answer]
Oops, if it's only a missing line, i will fix it this night, thanks for the info

**Reply by lampersky** [reply]
I will check if any other is missed too. Thanks @jtkech

**Reply by lampersky** [reply]
I've just double checked, and only **BaseUrl** is missing.

**Reply by jtkech** [reply]
Thanks for confirming
